database/frontend/images.json

```diff
@@ -25,5 +25,20 @@
         "url": "https://kaboompics.com/",
         "category": "frontend",
         "subcategory": "images"
+    },
+    {
+        "name": "Picjumbo",
+        "description": "Download free stock photos, backgrounds and free high-resolution images for personal and commercial use",
+        "url": "https://picjumbo.com/",
+        "category": "frontend",
+        "subcategory": "images"
+    },
+    {
+        "name": "Shotstash",
+        "description": "Free HD Stock Pictures for Creative Professionals",
```

Comment:
hi, could you please add a little more description for it?

Reply:
I agree with @rupali-codes @swissjake. Maybe you can describe its advantages?